fields.yml

```diff
@@ -126,16 +126,31 @@
           description: >
             Region in which this host is running.
     
-        - name: instance.id
-          type: keyword
-          example: i-1234567890abcdef0
-          description: >
-            Instance ID of the host machine.
+        - name: instance
+          type: group
```

Comment:
The group feature does not work in ECS. You have to use `instance.id` and `instance.name`.